docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -47,7 +47,7 @@ Inside `kotlin {}`, you can write the following blocks:
 
 A _target_ is a part of the build responsible for compiling, testing, and packaging a piece of software aimed at
 one of the supported platforms. Kotlin provides targets for each platform, so you can instruct Kotlin to compile code for
-that specific target. To learn more about how to configure these targets, see [Set up targets for Kotlin Multiplatform](multiplatform-set-up-targets.md).
+that specific target. To learn more about how to configure these targets. See how to [set up a target](multiplatform-discover-project.md#targets).
```

Comment:
Did you mean to delete "To learn more about how to configure these targets." ? 🤔

Reply:
Yeah, missed extra sentence